spark/src/test/scala/com/alibaba/graphar/TestWriter.scala

```diff
@@ -17,6 +17,7 @@ package com.alibaba.graphar
 
 import com.alibaba.graphar.utils.IndexGenerator
 import com.alibaba.graphar.writer.{VertexWriter, EdgeWriter}
+import com.alibaba.graphar.utils
```

Comment:
modify this line to:
import com.alibaba.graphar.utils.FileSystem
or remove:
import com.alibaba.graphar.utils.IndexGenerator